docs/src/main/asciidoc/spring-cloud-consul.adoc

```diff
@@ -73,7 +72,7 @@ CAUTION: If you use <<spring-cloud-consul-config,Spring Cloud Consul Config>>, t
 
 The default service name, instance id and port, taken from the `Environment`, are `${spring.application.name}`, the Spring Context ID and `${server.port}` respectively.
 
-`@EnableDiscoveryClient` make the app into both a Consul "service" (i.e. it registers itself) and a "client" (i.e. it can query Consul to locate other services).
+To disable the auto registration of the client with Consul you can set `spring.cloud.service-registry.auto-registration.enabled` to `false`.
```

Comment:
There is also a property to disable the client as well, not just registration.

Reply:
Are you talking about in consul specifically or in commons?